extensions/visualizations/oppia-visualization-click-hexbins.directive.html

```diff
@@ -0,0 +1,28 @@
+<div id="click-hexbin-chart">
```

Comment:
Typically we don't use IDs. We use classes. Please try to follow existing conventions.

Reply:
Done.